src/App.js

```diff
@@ -32,6 +25,16 @@ function CurrentUser({ children }) {
 }
 
 export default function App() {
+  let location = useLocation();
+
+  useEffect(() => {
```

Comment:
Not blocking but thinking this may fit better in the AppRoutes file so we can keep the App definition purely declarative markup.

Reply:
Great point, i can move it anywhere that's as high up the root as possible, routes file should work fine! I'll get it done this morning and merge.